Startup.cs

```diff
@@ -25,23 +26,47 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
-            
+
+
+
+
             services.AddSingleton (Configuration);
             services.AddSingleton (typeof (IRedisService), typeof (RedisService));
             services.AddSingleton (typeof (ITransifexService), typeof (TransifexService));
             services.AddSingleton (typeof (IMongoDbService), typeof (MongoDbService));
+
+            //Swagger help page
+            //https://docs.microsoft.com/en-us/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio
+            // Register the Swagger generator, defining one or more Swagger documents
+            //entrar a http://localhost:[random port]/swagger/
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new Info { Title = "My API", Version = "v1" });
+            });
+
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
+
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
             app.UseStaticFiles();
 
+            // Enable middleware to serve generated Swagger as a JSON endpoint.
+            app.UseSwagger();
+
+            // Enable middleware to serve swagger-ui (HTML, JS, CSS, etc.), specifying the Swagger JSON endpoint.
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
+            });
+
+
             app.UseMvc (routes => {

```

Comment:
this should be replaced with `app.UseMvc()` if we're using `[Route]` on our actions (so that we don't specify routes twice)